src/style.css

```diff
@@ -1 +1,84 @@
 /* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+}
+
+body {
+  margin: 0;
+  box-sizing: border-box;
```

Comment:
Set this property in `* {...} ` selector. Read more - https://css-tricks.com/box-sizing/